packages/xo-server/src/index.js

```diff
@@ -283,12 +283,14 @@ async function registerPlugin(pluginPath, pluginName) {
   } = plugin
   let instance
 
+  const config = this._config
   const handleFactory = factory =>
     typeof factory === 'function'
       ? factory({
+          config: config.plugins?.[pluginName],
```

Comment:
I wonder if config or configuration is a good name for it, because there is already a dynamic configuration in the plugin API.

Reply:
The dynamic configuration is passed as first param to the `configure` method, it's a little bit isolated to this configuration. 

What about `config` -> `staticConfig` || `defaultConfig`?

Reply:
`staticConfig` is nice and different enough.